mesh/mesh.go

```diff
@@ -207,29 +211,52 @@ func (m *Mesh) GetLayer(index types.LayerID) (*types.Layer, error) {
 	return l, nil
 }
 
-func (m *Mesh) HandleLateBlock(b *types.Block) {
-	m.Info("Validate late block %s", b.Id())
-	oldPbase, newPbase := m.MeshValidator.HandleLateBlock(b)
-	m.pushLayersToState(oldPbase, newPbase)
-	if err := m.PersistTortoise(); err != nil {
-		m.Error("could not persist Tortoise on late block %s from layer index %d", b.Id(), b.Layer())
-	}
+type validator struct {
+	*Mesh
+	processedLayer types.LayerID
+}
+
+func (m *validator) ProcessedLayer() types.LayerID {
+	defer m.lvMutex.RUnlock()
+	m.lvMutex.RLock()
+	return m.processedLayer
 }
 
-func (m *Mesh) ValidateLayer(lyr *types.Layer) {
-	m.Info("Validate layer %d", lyr.Index())
-	oldPbase, newPbase := m.HandleIncomingLayer(lyr)
+func (m *validator) SetProcessedLayer(lyr types.LayerID) {
+	m.Info("processed layer %d set processed layer %d", m.processedLayer, lyr)
```

Comment:
I don't understand this log message. Do you mean: previous processed layer is %d,; setting it to %d?

Reply:
debug log committed by mistake